start.cmd

```diff
@@ -75,7 +75,7 @@ for /f "tokens=2*" %%a in ('reg query "HKCU\Software\Microsoft\Windows\CurrentVe
 @rem TODO: External proxy when ORIG_PROXY_ENABLE == 0x1
 echo set ws = createobject("wscript.shell") > "%temp%\proxy.vbs"
 echo ws.currentdirectory = "%MITMDUMP_PATH%" >> "%temp%\proxy.vbs"
-echo ws.run "cmd /c mitmdump.exe -s "^&chr(34)^&"%PROXY_SCRIPT_NAME%"^&chr(34)^&" -k",0 >> "%temp%\proxy.vbs"
+echo ws.run "cmd /c mitmdump.exe -s "^&chr(34)^&"%PROXY_SCRIPT_NAME%"^&chr(34)^&" -k --allow-hosts \".*\.yuanshen\.com|.*\.mihoyo\.com|.*\.hoyoverse\.com\"",0 >> "%temp%\proxy.vbs"

```

Comment:
You can't use escape symbols in vbs to handle quote nesting, use `"^&chr(34)^&"` instead.